app/tests/test_record_metadata.py

```diff
@@ -0,0 +1,17 @@
+import re
+from playwright.sync_api import Page, expect
+
+
+def test_page_title_and_header(page: Page):
+    page.goto("http://localhost:5000/record")
+    expect(page).to_have_title(
```

Comment:
could be worth installing pytest-bdd so that we can use gherking syntax and reuse building blocks.

example: https://github.com/cmoir/playwright_pytest_bdd_example/blob/master/tests/test_with_bdd.py

But not for now